src/routes/init/import.ts

```diff
@@ -0,0 +1,346 @@
+import fetch from 'node-fetch';
```

Comment:
Could you move `import.ts`,`import.spec.ts`, and `interfaces.ts` into a subdirectory `src/routes/init/db` and change the names from `import` to the `index` naming convention?